Sources/VideoOptionsControlBar.swift

```diff
@@ -24,63 +24,84 @@ protocol VideoOptionsControlBarDelegate: class {
     // MARK: Instance variables
     weak var delegate: VideoOptionsControlBarDelegate?
     
+    var repeatMode: VLCRepeatMode {
+        didSet {
+            switch repeatMode {
+            case .repeatCurrentItem:
+                repeatButton.setImage(UIImage(named: "repeatOne-new"), for: .normal)
+            case .repeatAllItems:
+                repeatButton.setImage(UIImage(named: "repeat-new"), for: .normal)
+            default: // no repeat
+                repeatButton.setImage(UIImage(named: "no-repeat-new"), for: .normal)
```

Comment:
I'd rather handle the no repeat case separately and in default add in an assertionfailure("unhandled repeatmode"). That way if we edit repeat mode we immediately know that we're not handling something